src/Apps/ArtAdvisor/07-Curated-Discovery/llm/extractBudgetIntent.ts

```diff
@@ -0,0 +1,85 @@
+import { generateObject } from "ai"
+import { openai } from "@ai-sdk/openai"
+import dedent from "dedent"
+import { z } from "zod"
+
+const model = openai("gpt-4o")
```

Comment:
The brand new Claude Sonnet 3.5 model appears to be [the new leader on many benchmarks](https://arstechnica.com/information-technology/2024/06/anthropics-latest-best-ai-model-is-twice-as-fast-and-still-terrible-at-dad-jokes/), but so far I observed 4o doing better on this task. Can maybe revisit.

Reply:
Nice. Will be interesting to test these more thoroughly for our use cases.